### DISCUSSION TITLE
Customize transitions

### DISCUSSION THREAD

**Discussion by YasSerhbh** [shuding/next-view-transitions#45] [Q&A]
Hey, can anyone tell me how can I cutomize transitions ?

**Comment by neimannico** [comment]
Hey @YasSerhbh , you can refer to the [view-transition-name](https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Properties/view-transition-name) documentation to customize the transitions.
More details can also be found on the main [View Transitions API documentation](https://developer.mozilla.org/en-US/docs/Web/API/View_Transition_API/Using#customizing_your_animations) page.
